test/utils/uniswap.ts

```diff
@@ -141,8 +141,8 @@ async function addLiquidity(
   const timestampBefore = blockBefore.timestamp;
 
   await nonFungiblePositionManager.mint({
-    token0: paymentToken.address,
-    token1: linkToken.address,
+    token0: linkToken.address,
+    token1: paymentToken.address,
```

Comment:
It's because of how Uniswap calculates the pool ID using the token addresses values: https://github.com/Uniswap/v3-periphery/blob/main/contracts/NonfungiblePositionManager.sol#L128